dask/utils.py

```diff
@@ -1053,3 +1053,14 @@ def ensure_dict(d):
             result.update(dd)
         return result
     return dict(d)
+
+
+def package_of(obj):
+    """ Return package containing object's definition
+
+    Or return None if not found
+    """
+    # http://stackoverflow.com/questions/43462701/get-package-of-python-object/43462865#43462865
+    mod = inspect.getmodule(obj)
+    base, _sep, _stem = mod.__name__.partition('.')
```

Comment:
I may be doing something wrong, but looks like this errors for numpy arrays or anything where `inspect.getmodule` returns `None`:

```python
In [24]: x = np.ones(1)

In [25]: package_of(x)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-25-427cf35109bf> in <module>()
----> 1 package_of(x)

<ipython-input-14-209ff1f7b671> in package_of(obj)
      5     # http://stackoverflow.com/questions/43462701/get-package-of-python-object/43462865#43462865
      6     mod = inspect.getmodule(obj)
----> 7     base, _sep, _stem = mod.__name__.partition('.')
      8     return sys.modules[base]

AttributeError: 'NoneType' object has no attribute '__name__'
```